apiary.apib

```diff
@@ -3,245 +3,212 @@ HOST: http://lore.mit.edu/
 
 # LORE
 
-Simple API on top of MIT LORE application
+A REST API on top of MIT LORE application.
```

Comment:
Consider adding this text to explain what the API is for.  This first paragraph is from the LORE spec.

The Learning Objects Repository for Education (LORE) project seeks to build a library of online “learning objects” that will enable the efficient reuse of resources from previous MITx courses by professors and instructional staff.  This will enable (a) importing resources from earlier courses along with context and data about its use, (b) curating of new and previously used content, and (c) gathering metadata about content, including problem efficacy, and learning objectives for parts of the course content.  This repository will primarily serve MIT faculty and instructor teams delivering residential MITx and MITx on edX courses.
This API exposes the LORE application as a RESTful web service.